en/components/tabs.md

```diff
@@ -184,6 +184,201 @@ If changing the tabs' labels and icons is not enough, you can also create your o
 </igx-tabs>
 ```
 
+### Using Tabs and Routing
+
+The following examples demonstrate sample usage of the tabs component and basic routing scenarios. You can learn more about Angular Routing & Navigation <a href="https://angular.io/guide/router" target="_blank">here</a>. 
+
+#### Using igxTab, routerLink Directives and Single router-outlet
+
+In order to implement basic routing with **igx-tabs**, you can re-template the igx-tabs item header using the `igxTab` directive and provide links via `routerLink` in `ng-template`. Views are switched within a single `router-outlet` placed outside the tabs component. Note that `ng-template` content overides the default tabs headers style.
+
+```html
+<!-- tabs-sample-1.component.html -->
+<igx-tabs #tabs1>
+  <igx-tabs-group *ngFor="let routerLink of routerLinks">
+    <ng-template igxTab>
+      <a routerLink="{{routerLink.link}}">
+        {{routerLink.label}}
+      </a>
+    </ng-template>
+  </igx-tabs-group>
+</igx-tabs>
+
+<div>
+  <router-outlet></router-outlet>
+</div>
+```
+
+```typescript
+// tabs-sample-1.component.ts
+this.routerLinks = [
+  {
+    label: 'View 1',
+    link: '/view1',
+    index: 0
+  }, 
+  {
+    label: 'View 2',
+    link: '/view2',
+    index: 1
+  },
+  {
+    label: 'View 3',
+    link: '/view3',
+    index: 2
+  },
+];
+```
+Declare all needed route definitions that map URL path to a specific component:
+
+```typescript
+// app.routing.module.ts
+const routes: Routes = [
+    // simple links
+    { path: '', redirectTo: 'view1', pathMatch: 'full' },
+    { path: 'view1', component: View1Component },
+    { path: 'view2', component: View2Component },
+    { path: 'view3', component: View3Component },
+];
+
+@NgModule({
+    imports: [
+        RouterModule.forRoot(routes)
```

Comment:
In my opinion a better use case scenario will be define those child routes in another router module and use `RouterModule.forChild`.

Reply:
https://angular.io/guide/router#hero-feature-route-configuration